src/main/sensors/voltage.c

```diff
@@ -201,7 +206,11 @@ void voltageMeterADCRefresh(void)
 void voltageMeterADCRead(voltageSensorADC_e adcChannel, voltageMeter_t *voltageMeter)
 {
     voltageMeterADCState_t *state = &voltageMeterADCStates[adcChannel];
-
+    
+    if (millis() - voltageMeter->prevDisplayTime > PREV_DISPLAY_FILTERED_TIME_DIFF) {
```

Comment:
IMO batteryUpdateVoltage is better place for this.,

Also, cmp32 for timestamp with overflow (I know that 49days is a lot of time. Abut eventually someone will get bitten by it)

Reply:
changed